examples/example_mig_juiceshop/dev/README.md

```diff
@@ -0,0 +1 @@
+## First Commit: In this folder we will have examples related to the Managed Instance Group
```

Comment:
Can you rename the top level example folder to `web-app-protection-example`

Reply:
Sure.